PowerUp/app/src/main/java/powerup/systers/com/kill_the_virus_game/KillTheVirusEndActivity.java

```diff
@@ -19,11 +21,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_kill_the_virus_end);
         ButterKnife.bind(this);
-        //TODO: Update the score text view to display scored points
+        updateScore();
     }
 
     @OnClick
     public void clickContinue() {
-        //TODO: Migrate to scene completion screen
```

Comment:
Continue button is not properly implemented, it's not working at the moment. Bind the continue button to the method.